openreview/venue/process/invitation_edit_process.py

```diff
@@ -90,7 +101,8 @@ def update_note_readers(submission, paper_invitation):
                         note = openreview.api.Note(
                             id = note.id,
                             readers = final_invitation_readers,
-                            nonreaders = paper_invitation.edit['note'].get('nonreaders')
```

Comment:
@melisabok should I post the edit only with what is changing here too?
For example, if the readers of the note are not changing, should the edit not contain the readers?

Reply:
that's correct, we shouldn't pass the values that don't change.